conventions/documentation/quick-starts.md

```diff
@@ -190,13 +184,15 @@ If the user answers No, an error message appears with a link to relevant documen
   * **Team responsible:** Product Management; UXD Design; possibly UXD Research
   * **Time frame:** Before epic prioritization
   * **Definition of done:** Product Management, UXD Design, UXD Research, and any other groups in this step agree on the product area to prioritize for a quick start. Epics are created in the relevant development project.
-2. Plan user flows and interaction design. (Optional)
+
+Step 2:
+Plan user flows and interaction design.
   * UXD Designer, UXD Content Strategist, and CCS Content Strategist and/or Writer meet with Product Manager to determine the quick start’s goal and where we need to reduce friction or confusion in completing some tasks.
   * Collaborate on what the ideal user flow would look like. For example, what are the tasks and the sub tasks that users need to do in order to complete the quick start?
   * Work with Abi to figure out who we work with in CCS and tag in JIRA story.
     * Track prioritizations in JIRA.
     * UXD Content and CCS may not have bandwidth to review quick starts for external teams. If a full editorial review isn't possible, they might be able to provide a quick review and high-level feedback, or the external team can do a self-review using the writing guidelines in this doc as a reference.
-  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs.
+  * At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs. (Optional)
```

Comment:
I'm thinking it might make sense to move "(Optional)" at the front of this step like this: 

```suggestion
  * (Optional) At this point, the UXD Designer is putting designs into Sketch and Marvel and getting feedback on interactions (not content yet). These will not be hi-fi designs. 
```

Reply:
Done!